sos/report/plugins/origin.py

```diff
@@ -107,7 +107,8 @@ def setup(self):
             if self.is_static_etcd():
                 self.add_cmd_output("%s etcd etcd" % static_pod_logs_cmd)
 
-            # TODO: some thoughts about information that might also be useful
+            # Possible enhancements:
```

Comment:
This doesn't feel like a TODO. So reworded it.

If this does need to be tracked, I can open a new issue and link here.